aspect/intellij_info_impl.bzl

```diff
@@ -354,7 +354,14 @@ def _collect_generated_go_sources(target):
         files = target[OutputGroupInfo].go_generated_srcs.to_list()
         return [f for f in files if f.basename.endswith(".go")]
     else:
-        return None
+        files = target.files.to_list()
```

Comment:
Yes, when we use vscode with rules_go/gopackagesdriver,there is a problem that the source will include some .s file.This will cause the indexing broken. Let me try to setting `go_generated_srcs`.

Reply:
See at line 364, we only collect the file to avoid broke indexing. In the latest go rules, it use the same judging condition.

Reply:
93mai In which as Which one in Whole?